pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/AuthenticationRoute.java

```diff
@@ -121,7 +125,7 @@ public Object handle(Request request, Response response) {
                             haltWithErrorMsg(401, response, "DSMToken was null! Not authorized user", e);
                         }
                     } else {
-                        haltWithErrorMsg(400, response, "user was null");
+                        haltWithErrorMsg(400, response, "user info in token was null");
```

Comment:
this class will rot, we can survive if we change nested if to guard conditions like:
`if (StringUtils.isBlank(auth0Token)) { haltWithErrorMsg(400, response, "user info in token was null"); }`
such structure to avoid nested if statements